### PR TITLE
I wasn't high when I wrote this.

### PR DESCRIPTION
**Pull request by stevegraham**
Allow "illegal" postcodes for addresses outside the UK

### CODE REVIEW

File: app/forms/claimant_form.rb [hunk 1]
```diff
@@ -6,7 +6,11 @@ class ClaimantForm < Form
   EMAIL_ADDRESS_LENGTH = 100
   NAME_LENGTH          = 100
 
-  include AddressAttributes
+  include AddressAttributes.but_skip_postcode_validation
```
Comment (stevegraham) [inline app/forms/claimant_form.rb]: Literate programming FTW. :no_smoking: :dizzy: :dizzy_face:

File: app/forms/claimant_form.rb [hunk 2]
```diff
@@ -32,8 +36,10 @@ class ClaimantForm < Form
   validates :contact_preference, inclusion: { in: CONTACT_PREFERENCES }
   validates :mobile_number, :fax_number, length: { maximum: PHONE_NUMBER_LENGTH }
   validates :address_country, inclusion: { in: COUNTRIES }
-  validates :fax_number,    presence: { if: -> { contact_preference.fax? } }
-  validates :email_address, presence: { if: -> { contact_preference.email? } }, length: { maximum: EMAIL_ADDRESS_LENGTH }
+  validates :fax_number,    presence: { if: :contact_preference_fax? }
+  validates :email_address, presence: { if: :contact_preference_email? }, length: { maximum: EMAIL_ADDRESS_LENGTH }
+
+  delegate :fax?, :email?, to: :contact_preference, prefix: true
```
Comment (stevegraham) [inline app/forms/claimant_form.rb]: This is just pure code climate assuagement.

File: app/forms/claimant_form.rb (hunk 1 above)
Comment (dan-palmer) [inline app/forms/claimant_form.rb]: :tophat:

**Comment by dan-palmer** [comment]
:rocket: :roller_coaster: :ship:

**Comment by stevegraham** [comment]
Cheers @dan-palmer

**Pull request merged**